apps/menu-bar/package.json

```diff
@@ -20,26 +20,26 @@
     "react-native-vector-icons": "^9.2.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.0",
-    "@babel/preset-env": "^7.20.0",
-    "@babel/runtime": "^7.20.0",
-    "@react-native-community/eslint-config": "^3.2.0",
-    "@tsconfig/react-native": "^2.0.2",
+    "@babel/core": "^7.22.11",
+    "@babel/preset-env": "^7.22.10",
+    "@babel/runtime": "^7.22.11",
+    "@tsconfig/react-native": "^2.0.3",
     "@types/jest": "^29.2.1",
     "@types/react": "^18.0.24",
     "@types/react-native-vector-icons": "^6.4.13",
     "@types/react-test-renderer": "^18.0.0",
-    "@typescript-eslint/eslint-plugin": "^6.1.0",
-    "@typescript-eslint/parser": "^6.1.0",
     "babel-jest": "^29.2.1",
-    "eslint": "^8.45.0",
+    "eslint": "^8.48.0",
+    "eslint-config-universe": "^12.0.0",
     "jest": "^29.2.1",
     "metro-react-native-babel-preset": "0.73.9",
-    "prettier": "^2.4.1",
-    "prettier-eslint": "^15.0.1",
+    "prettier": "^3.0.2",
     "react-native-svg-transformer": "^1.0.0",
     "react-test-renderer": "18.2.0",
-    "typescript": "4.8.4"
+    "typescript": "5.1.6"
+  },
+  "resolutions": {
```

Comment:
Why do we need to use `resolutions`?

Reply:
Nice catch! Nope, since James bumped the plugin package to version supporting Prettier 3 in latest major release.